File: src/main/java/com/github/e2318501/anyban/BannedIp.java
package com.github.e2318501.anyban;

import lombok.Getter;
import lombok.RequiredArgsConstructor;

@Getter
@RequiredArgsConstructor
public class BannedIp {
    private final String ip;
    private final String created;
    private final String source;
    private final String expires;
    private final String reason;
}

File: src/main/kotlin/com/github/tsuoihito/anyban/model/BannedIp.kt
package com.github.tsuoihito.anyban.model

import com.fasterxml.jackson.annotation.JsonProperty

data class BannedIp(
    @JsonProperty val ip: String,
    @JsonProperty val created: String,
    @JsonProperty val source: String,
    @JsonProperty val expires: String,
    @JsonProperty val reason: String
)
